docker/cleanspeak/release-trigger

```diff
@@ -1,3 +1,3 @@
-version=3.33.0
+version=4.0.0
```

Comment:
I don't think you want to change this trigger file. This will trigger the release of this image, but the bundles are not published, so I would expect this to fail.

Reply:
yes the PR description kind of alludes to this. I can see now that if we don't change this file, then we should avoid triggering the workflow so I will back this out.